core/persistence-jpa/src/main/java/org/apache/syncope/core/persistence/jpa/dao/JPAAnySearchDAO.java

```diff
@@ -769,7 +783,12 @@ private void fillAttributeQuery(final StringBuilder query, final PlainAttrValue
                     if (not) {
                         query.append(" NOT ");
                     }
-                    query.append(" LIKE ?").append(setParameter(parameters, cond.getExpression()));
+                    query.append(" LIKE ");
```

Comment:
LOWER function is defined in SQL 2003.
http://users.atw.hu/sqlnut/sqlnut2-chp-4-sect-4.html#sqlnut2-CHP-4-TABLE-9

(Sorry I didn't found a better reference)